apps/xomg_tasks/lib/utils.ex

```diff
@@ -21,7 +21,7 @@ defmodule XomgTasks.Utils do
   Runs a specific app for some arguments. Will handle IEx, if one's running
   """
   def generic_run(args, apps) when is_list(apps) do
-    Mix.Task.run("run", args)
+    XomgTasks.Run.run(args)
```

Comment:
Looks like `XomgTasks.Run.run/1` is only printing the args back to the shell. Maybe that module and this line are no longer needed?

Reply:
indeed!